### PR TITLE
Fixes for our non-latin characters issues. This one tem…

### PR DESCRIPTION
**Pull request by Dc41f**
…porairly fixes the HIG component crashing when using non-simple-latin characters in project label and adds documentation entry on how to properly create UTF-8 encoded zip file. Im still testing different scenarios and i will be working on preventing the adoption of  project with file names containing characters outside of the Unicode set - this scenaril will end up with an error stating tat the name was wrong and it will contain link to the new readme entry about zip and UTF-8 encoding.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,6 +170,17 @@ We are using npm.
     * First change `WebApplication.Program.cs` by removing the `UseKestrel()` statement
     * You will need to change the callback in your forge app to match the URL you deploy to.
 	
+### Project file zip encoding
+* Uploading of project file can end up with error "zip file was not encoded properly". This problem typicaly hapens when you use non latin character in your project file names and the zip file is not using UTF-8 encoding for the project file names.
```
Comment (erymski) [inline README.md:174]:
```suggestion
* Uploading of project file can end up with error "zip file was not encoded properly". This problem typically happens when you use non latin character in your project file names and the zip file is not using UTF-8 encoding for the project file names.
```

File: README.md [hunk 2]
```diff
@@ -170,6 +170,17 @@ We are using npm.
     * First change `WebApplication.Program.cs` by removing the `UseKestrel()` statement
     * You will need to change the callback in your forge app to match the URL you deploy to.
 	
+### Project file zip encoding
```
Comment (erymski) [inline README.md:173]: I think link to the [FDA troubleshooting page](https://forge.autodesk.com/en/docs/design-automation/v3/developers_guide/troubleshooting/) also can be useful here.

Comment (Dc41f) [reply README.md:173]: Good point, i have added the link to the paragraph :)

File: README.md [hunk 3]
```diff
@@ -170,6 +170,19 @@ We are using npm.
...
+* Uploading of project file can end up with error "Project name or asembly contains unsupported characters". This problem typically happens when you use non latin character in your project file names and the zip file is not using UTF-8 encoding for the project file names.
```
Comment (andersco) [inline README.md:174]:
```suggestion
* Uploading your project file can cause the error "Project name or assembly contains unsupported characters". This problem typically happens when you use non Latin characters in your project file names and the zip file does not use UTF-8 encoding for the project file names.
```

File: README.md [hunk 4]
```diff
@@ -170,6 +170,19 @@ We are using npm.
...
+* This issue can be fixed by forcing your zip program of choice to zip the file with the UTF-8 encoding
+* For 7-zip, you can follow these simple steps in order to get your valid project file zip:
```
Comment (andersco) [inline README.md:176]:
```suggestion
* For 7-zip, you can follow these simple steps:
```

File: README.md [hunk 5]
```diff
@@ -170,6 +170,19 @@ We are using npm.
...
+	* After selecting your files for zipping, press right mouse button and select 7-zip from the context menu
```
Comment (andersco) [inline README.md:177]:
```suggestion
	* After selecting your files for zipping, right-click them and select 7-zip from the context menu
```

File: README.md [hunk 6]
```diff
@@ -170,6 +170,19 @@ We are using npm.
...
+	* Select option "Add to archive..."
+	* On the very bottom you can see text field "Parameters:"
```
Comment (andersco) [inline README.md:179]:
```suggestion
	* On the very bottom you can see a text field labelled "Parameters:"
```

File: README.md [hunk 7]
```diff
@@ -170,6 +170,19 @@ We are using npm.
...
+	* Write "cu" without the quotes into that field
+	* Press ok
+	* Name your resulting zip
```
Comment (andersco) [inline README.md:182]:
```suggestion
	* Name your resulting zip file
```

File: README.md [hunk 8]
```diff
@@ -170,6 +170,19 @@ We are using npm.
...
+	
+* For more information on this issue please refer to [FDA troubleshooting page](https://forge.autodesk.com/en/docs/design-automation/v3/developers_guide/troubleshooting/)
```
Comment (andersco) [inline README.md:184]:
```suggestion
* For more information on this issue please refer to the [FDA troubleshooting page](https://forge.autodesk.com/en/docs/design-automation/v3/developers_guide/troubleshooting/)
```

**Review by andersco — APPROVED** [review]

**Review by mivasi — APPROVED** [review]

**Pull request merged**